packages/avatars/demo/stories/StatusIndicatorStory.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story } from '@storybook/react';
+import { StatusIndicator, IStatusIndicatorProps } from '@zendeskgarden/react-avatars';
+
+interface IArgs extends IStatusIndicatorProps {
+  caption?: string;
+  show?: string;
+}
```

Comment:
To repeat, this data should simply be based on `children`. The custom `IArgs` and `renderCaption` are probably not necessary. See https://github.com/zendeskgarden/react-components/blob/main/packages/tags/demo/stories/TagStory.tsx